### ISSUE TITLE
Error calling method from signal 'changed'

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#4745]
**Operating system or device - Godot version:**
compiled 21f965f249799d5f6adb607a6ea5a4cd728f28b0

**Issue description** (what happened, and what was expected):
Error message comes up when using Material on CanvasItem nodes.
```
Error calling method from signal 'changed': 'CanvasItemMaterial::_shader_changed': Method not found.
```

**Steps to reproduce:**
1. Add a Control or Node2D node to scene
2. Make a new CanvasItemMaterial of node at Material
3. Edit CanvasItemMaterial
4. Make a new CanvasItemShader or CanvasItemShaderGraph
5. Edit Shader
6. Try to do something with Shader

**Link to minimal example project** (optional but very welcome):

**Comment by MarianoGnu** [comment]
I confirmed the method _shader_changed exists in canvas_item.cpp, but it's not binded anywhere. In any case this method is empty and probably unused.

@reduz  if you belive this method is useless i'll delete it, otherwise point me what it's suposed to do to properly fix it.

**Comment by MarianoGnu** [comment]
Fixed in #5308

**Comment by TheSymbolSystem** [comment]
What does the Method Not Found error mean?
